_posts/2016-09-14-supercharged-seo-with-drupal-8.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "Supercharged SEO with Drupal 8"
+date: 2016-09-14
+author: Oksana Cyrwus
+tags: seo drupal drupal8
+summary: SEO trends 2017 and Drupal 8 as the best option for your long-term marketing campaign.
+featured_image: "/assets/img/blog/supercharged-seo-with-drupal-8.jpg"
+featured_image_alt: "Supercharged SEO with Drupal 8"
+---
+
+Here at Savas Labs, when it comes to developing systems, our focus is not only to code according to best practices, but also deliver a solid marketing vehicle to our clients and partners. Let’s be honest here - perfectly coded, a pretty looking website will be of little use if it doesn’t produce leads / facilitate conversions or generate revenue! So how do we achieve those goals? We get there by balancing website objectives while giving priority to lead generation via SEO. The more quality traffic that comes to the website - the more conversions we can achieve. It’s that simple!
```

Comment:
This whole paragraph is a great point. However it feel's it represents us in a limited way. I read "We do code well" and "we do marketing well". My thoughts are that we provide a wide variety of services, some of which are complex code delivery, some of which enable marketing needs, but that's not it. Can you adjust the paragraph a bit and phrase it in more of a "We listen to our clients needs, and what many of our client need are to reach their audiences effectively -- that's where SEO comes in"?